src/Symfony/Component/CssSelector/XPath/XPathExpr.php

```diff
@@ -46,9 +46,10 @@ public function getElement(): string
     /**
      * @return $this
      */
-    public function addCondition(string $condition): static
+    public function addCondition(string $condition/* , string $conjunction = 'and' */): static
```

Comment:
This class is internal as well, can't we add the new argument directly?

Reply:
Internal classes are not covered by the BC policy, so we can add it.

Reply:
Great. Update done.